FlexMeasures/flexmeasures
# Status page jobs updates

## Description

- Merged 2 tables for scheduling and forecasting jobs to a single one
- Added 'Queue' column
- Renamed 'Asset id' column to 'Entity' and added entity name + entity type
- Moved job_id to metadata
- Added "Info" button for each row, clicking on which results in copying job metadata to a clipboard

## Look & Feel

2 tables are merged into 1, info button is added

## How to test

Add some scheduling / forecasting jobs, make some of them fail
Check you get both job types in new table, check clicking on "Info" button copies this job metadata

## Further Improvements

-

## Related Items

-

---
